go/pkg/bertymessenger/util.go

```diff
@@ -2,17 +2,23 @@ package bertymessenger
 
 import (
 	"bytes"
+	"context"
 	"encoding/base64"
 
+	"berty.tech/berty/v2/go/pkg/bertyprotocol"
 	"berty.tech/berty/v2/go/pkg/bertytypes"
 	"berty.tech/berty/v2/go/pkg/errcode"
 )
 
 func (svc *service) checkIsMe(gme *bertytypes.GroupMessageEvent) (bool, error) {
```

Comment:
ambiguous, can you keep only one?

Reply:
Yes I hesitated while doing it, I will change it ;)